src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -335,5 +335,19 @@ private void DisableTrustWarningsChecked(object sender, RoutedEventArgs e)
                 viewModel.DisableTrustWarnings = (bool)(sender as ToggleButton).IsChecked;
             }
         }
+
+        private void ScrollViewer_PreviewMouseWheel(object sender, MouseWheelEventArgs e)
```

Comment:
Interesting, isn't this functionality defined by WPF by default for us? Just curious

Reply:
Yeah, it's weird because the action of the mouse wheel is being triggered but it's not scrolling without forcing it. I found this solution through this StackOverflow question: https://stackoverflow.com/questions/25366784/wpf-scrollviewer-mousewheel-not-working-with-stackpanel.

Reply:
I feel that is the case so I was hoping we can address on the xaml level but this is also fine. @filipeotero Do you mind take a look at the xaml to see if there is anything fishy? If none, we can just use this way

Reply:
Sure, I will take a look

Reply:
Hi @QilongTang, didn't find anything in the XAML that helps with the fix. Should we proceed in this way?